tests/dcps_tests.lst

```diff
@@ -153,8 +153,8 @@ tests/DCPS/NotifyTest/run_test.pl: !DCPS_MIN !GHA_OPENDDS_SAFETY_PROFILE
 tests/DCPS/Observer/run_test.pl: !DCPS_MIN !GH_ACTIONS
 tests/DCPS/Reliability/run_test.pl: !DCPS_MIN !DDS_NO_OWNERSHIP_PROFILE !OPENDDS_SAFETY_PROFILE
 tests/DCPS/Reliability/run_test.pl rtps: !DCPS_MIN !DDS_NO_OWNERSHIP_PROFILE
-tests/DCPS/Reliability/run_test.pl keep-last-one: !DCPS_MIN !OPENDDS_SAFETY_PROFILE
-tests/DCPS/Reliability/run_test.pl rtps keep-last-one: !DCPS_MIN !GH_ACTIONS
+tests/DCPS/Reliability/run_test.pl: !DCPS_MIN !OPENDDS_SAFETY_PROFILE
+tests/DCPS/Reliability/run_test.pl rtps: !DCPS_MIN !GH_ACTIONS
```

Comment:
See the two lines above these -- now these are duplicates.

Reply:
I thought they were different.

Compare Line 154 with Line 156: 
tests/DCPS/Reliability/run_test.pl: !DCPS_MIN !DDS_NO_OWNERSHIP_PROFILE !OPENDDS_SAFETY_PROFILE
tests/DCPS/Reliability/run_test.pl: !DCPS_MIN !OPENDDS_SAFETY_PROFILE

Compare Line 155 with Line 157: 
tests/DCPS/Reliability/run_test.pl rtps: !DCPS_MIN !DDS_NO_OWNERSHIP_PROFILE
tests/DCPS/Reliability/run_test.pl rtps: !DCPS_MIN !GH_ACTIONS